open-reac/src/main/java/com/powsybl/openreac/parameters/input/json/OpenReacParametersDeserializer.java

```diff
@@ -90,6 +90,22 @@ public OpenReacParameters deserialize(JsonParser parser, DeserializationContext
                     parser.nextToken();
                     parameters.setReactiveSlackBusesMode(ReactiveSlackBusesMode.valueOf(parser.getText()));
                     break;
+                case "defaultVariablesScalingFactor":
```

Comment:
And since you are already modifying this part of the code, you can refactor it and use the enhanced switch case. Here is an example from another PowSyBl repo, with the new syntax:

```
 switch (feederNode.getFeeder().getFeederType()) {
                    case BRANCH, TWO_WINDINGS_TRANSFORMER_LEG -> addOperatingStatusDecorator(nodeDecorators, node, direction, network.getBranch(feederNode.getEquipmentId()));
                    case THREE_WINDINGS_TRANSFORMER_LEG -> {
                        // if this is an outer leg (leg corresponding to another voltage level), we display the decorator on the inner 3wt
                        if (node.getAdjacentNodes().stream().noneMatch(Middle3WTNode.class::isInstance)) {
                            addOperatingStatusDecorator(nodeDecorators, node, direction, network.getThreeWindingsTransformer(feederNode.getEquipmentId()));
                        }
                    }
                    case HVDC -> addOperatingStatusDecorator(nodeDecorators, node, direction, network.getHvdcLine(feederNode.getEquipmentId()));
                    default -> { /* No decorator for other feeder types */ }
                }
```